components/scream/src/physics/p3/p3_functions_main_impl.hpp

```diff
@@ -1161,7 +1150,7 @@ ::p3_main(
     p3_main_pre_main_loop(
       team, nk, log_predictNc, dt,
       opres, opdel, odzq, onpccn, oexner, inv_exner, inv_lcldm, inv_icldm, inv_rcldm, oxxlv, oxxls, oxlf,
```

Comment:
Does *onpccn* need to be changed to *oncnuc*?

Reply:
Yeah, I think we should change npccn to ncnuc everywhere in p3_main_pre_main_loop.